target/classes/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.url= jdbc:postgresql://localhost:5432/testdb
```

Comment:
I went ahead and used Jasypt to encrypt "url", "username", and "password". Please let me know if this approach looks reasonable.

I have to now run the application with the command: `mvn -Djasypt.encryptor.password={ENCRYPTION_PASSWORD} spring-boot:run`

Reply:
I have now instead added the `jasypt.encryptor.password` property inside the properties file, whose value is taken from an env variable that is exported before running our application. So, now we run our application using the following command:

`export JASYPT_ENCRYPTOR_PASSWORD={ENCRYPTION_PASSWORD}`
`mvn spring-boot:run`

Reply:
Something like `spring.datasource.url=jdbc:postgresql://${POSTGRES_HOST:-localhost}:${POSTGRES_PORT:-5432}/${POSTGRES_DB:-health}` would have been better as it will be easier when it comes to deploy the application or if I want to run it on my side.